CommunityToolkit.Graph.Uwp/Extensions/ElementExtensions.cs

```diff
@@ -0,0 +1,120 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using CommunityToolkit.Authentication;
+using Windows.UI.Xaml;
+
+namespace CommunityToolkit.Graph.Uwp
+{
+    /// <summary>
+    /// Extensions on FrameworkElement for responding to authentication changes from XAML.
+    /// </summary>
+    public class ElementExtensions : DependencyObject
```

Comment:
Think this could be static as well? See what we do for other extensions/helpers in the Toolkit?